website/server/controllers/api-v3/pushNotifications.js

```diff
@@ -0,0 +1,108 @@
+import { authWithHeaders } from '../../middlewares/api-v3/auth';
+import _ from 'lodash';
+import {
+  BadRequest,
+  NotAuthorized,
+} from '../../libs/api-v3/errors';
+
+let api = {};
+
```

Comment:
No, it wasn't really my intention to allow it to be included in an email. I'll change it to DELETE.
I wouldn't need such an API call for the official clients. But if others think having a GET method to list all devices would make sense, I can add it.